hephaistus/main-theme/distribution-system/chord/playground/TheWeaVer/my-chord/src/main/java/com/lia/hephaistus/chord/theweaver/DefaultChordNode.java

```diff
@@ -0,0 +1,67 @@
+package com.lia.hephaistus.chord.theweaver;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The 128 size chord node(m = 7).
+ * It does not generate any key, but throws exception if the given key is lager than 128.
+ */
+public class DefaultChordNode implements ChordNode {
```

Comment:
[nit] If there is a functions to be implemented in later PR, then I think it is better to add some comment or some code like `throw Exception("Not yet implemented")`.

Reply:
Thank you, I apply it on my IDE
![image (1)](https://github.com/TheWeaVer/lia/assets/13251510/be0bffeb-1581-4796-9b02-f4366cde4ee8)